pkg/controllers/status/controller.go

```diff
@@ -422,32 +428,31 @@ func (s *KubeFedStatusController) reconcile(qualifiedName common.QualifiedName)
 			if apierrors.IsConflict(err) {
 				return worker.StatusConflict
 			}
-			runtime.HandleError(errors.Wrapf(err, "Failed to update status object for federated type %s %q", statusKind, key))
+			keyedLogger.Error(err, "Failed to update status object")
 			return worker.StatusError
 		}
 	}
 
 	return worker.StatusAllOK
 }
 
-func (s *KubeFedStatusController) rawObjFromCache(store cache.Store, kind, key string) (pkgruntime.Object, error) {
+func (s *StatusController) rawObjFromCache(ctx context.Context, store cache.Store, kind, key string) (pkgruntime.Object, error) {
```

Comment:
Does this method still need a `ctx`? Same for `objFromCache`

Reply:
In fact, `kind` is also redundant. Pls recursively clean up the redundant variables/parameters.